groundzero/src/components/NavigationBar.jsx

```diff
@@ -1,8 +1,41 @@
 import React from 'react'
+import { Link } from 'react-scroll'
+import "../index.css"
 
 const NavigationBar = () => {
   return (
-    <div>NavigationBar</div>
+    <nav className="nav">
+      <a href="/" className="site-title">[image]</a>
+      <ul>
+        <li>
+          <Link activeClass="active" to="about" spy={true} smooth={true} offset={-100} duration={500}>
+            About
+          </Link>
+        </li>
+        <li>
+          <Link activeClass="active" to="timeline" spy={true} smooth={true} offset={-100} duration={500}>
+            Timeline
+          </Link>
+        </li>
+        <li>
+          <Link activeClass="active" to="problems" spy={true} smooth={true} offset={-100} duration={500}>
+            Problems
+          </Link>
+        </li>
+        <li>
+          <Link activeClass="active" to="contact" spy={true} smooth={true} offset={-100} duration={500}>
+            Contact
+          </Link>
+        </li>
+        <li>
+          <Link activeClass="active" to="signup" spy={true} smooth={true} offset={-100} duration={500}>
```

Comment:
For this part I think can also make use of the constants file and then import and map the elements from there, since the styling for each element is kind of the same so we can decrease the code length :D